components/suggest/src/schema.rs

```diff
@@ -193,6 +196,46 @@ CREATE TABLE IF NOT EXISTS dismissed_suggestions (
                 )?;
                 Ok(())
             }
+            19 => {
+                // Clear the database since we're going to be dropping the keywords table and
+                // re-creating it
+                clear_database(tx)?;
+                tx.execute_batch(
+                    "
+-- Recreate the various keywords table to drop the `ON DELETE` clauses
+DROP TABLE keywords;
+DROP TABLE full_keywords;
+DROP TABLE prefix_keywords;
+CREATE TABLE keywords(
+    keyword TEXT NOT NULL,
+    suggestion_id INTEGER NOT NULL REFERENCES suggestions(id),
```

Comment:
Alternatively, we could also add an `record_id` here with an index on it so that we can delete all the keywords by a record_id. That should save a sub-query in the delete statement above. Though the upside with your approach is that it'll be easier to support delta updates as it only needs to delete old keywords for updated suggestions instead of all the suggestions for a RS record.

Reply:
Oooh, interesting! I'd be curious to see if this has any adverse impact on the file size.

Intuitively, I'm worried that duplicating (and indexing, which would duplicate it again) the `record_id` in the `keywords` table (for 180k rows), but maybe SQLite can actually store it efficiently.

Reply:
Good point. I didn't think of that overhead which indeed would bloat the db. It would only make sense for a numeric typed column.

Reply:
Okay, let's skip this for now since it doesn't seem like a clear win.

Reply:
Sounds good.